tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/SymbolAtCursorTest.java

```diff
@@ -191,13 +191,14 @@ public void testMissingConstructs(int line, int column) {
     }
 
     @Test(dataProvider = "QuotedIdentifierProvider")
-    public void testQuotedIdentifiers(int line, int column, String expSymbolName) {
+    public void testQuotedIdentifiers(int line, int column, String expSymbolName, String expSymbolNameUnQuoted) {
         Project project = BCompileUtil.loadProject("test-src/symbol_at_cursor_quoted_identifiers_test.bal");
         SemanticModel model = getDefaultModulesSemanticModel(project);
         Document srcFile = getDocumentForSingleSource(project);
 
         Optional<Symbol> symbol = model.symbol(srcFile, LinePosition.from(line, column));
-        symbol.ifPresent(value -> assertEquals(value.getName().get(), expSymbolName));
+        symbol.ifPresent(value ->
+                assertTrue(value.nameEquals(expSymbolName) && value.nameEquals(expSymbolNameUnQuoted)));
```

Comment:
Shall we add this as 2 different assert statements? So that if there's a failure we can easily recognize which comparison failed